app/src/main/java/org/fossasia/openevent/app/common/data/repository/CopyrightRepository.java

```diff
@@ -50,8 +50,7 @@ public CopyrightRepository(IUtilModel utilModel, IDatabaseRepository databaseRep
         Observable<Copyright> networkObservable = Observable.defer(() ->
             eventService.getCopyright(eventId)
                 .doOnNext(copyright -> databaseRepository
-                    .delete(Copyright.class, Copyright_Table.event_id.eq(eventId))
-                    .concatWith(databaseRepository.save(Copyright.class, copyright))
+                    .save(Copyright.class, copyright)
```

Comment:
@iamareebjamal I have a query. 
I am getting an ```onErrorNotImplemented``` exception if i don't change this line. 
I checked the ```TicketsRepository``` for reference and it doesn't delete the previous ```Ticket``` entries. 

``` Java
Observable<Ticket> networkObservable = Observable.defer(() ->
            eventService.getTicket(ticketId)
                .doOnNext(ticket -> databaseRepository
                    .save(Ticket.class, ticket)
                    .subscribe()));
``` 
Hence i have also changed to just save the ```Copyright``` entry without deleting the previous Copyright values.
Is it the correct way or should i open an issue for the TicketRepository ?